interview-questions.md

```diff
@@ -6,64 +6,88 @@ First, without external resources. Challenge yourself to answer from memory as i
 
 Then, research the question to expand on your answer. Even if you feel you have answered the question completely, there is always something more to learn. Write your researched answer in your OWN WORDS.
 
+
+
+
+
```

Comment:
As stated having a conversation vice just giving a quick answer is a beautiful approach to interviews as well as including your experience. Great job!